src/Sarif.FunctionalTests/ConverterTestData/Semmle/results.csv.sarif

```diff
@@ -0,0 +1,1353 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-1.0.0",
+  "version": "1.0.0",
+  "runs": [
+    {
+      "tool": {
+        "name": "Semmle"
+      },
+      "results": [
+        {
+          "level": "warning",
+          "message": "Avoid using thread priorities. The effect is unpredictable and not portable.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/FileCleaningTracker.java",
+                "region": {
+                  "startLine": 211,
+                  "startColumn": 13,
+                  "endLine": 211,
+                  "endColumn": 44
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "Incorrect lazy initialization of static field [[\"SKIP_BYTE_BUFFER\"|\"relative://src/main/java/org/apache/commons/io/IOUtils.java:170:6561:16\"]]: Missing synchronization.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/IOUtils.java",
+                "region": {
+                  "startLine": 2718,
+                  "startColumn": 13,
+                  "endLine": 2718,
+                  "endColumn": 57
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "Incorrect lazy initialization of static field [[\"SKIP_CHAR_BUFFER\"|\"relative://src/main/java/org/apache/commons/io/IOUtils.java:169:6517:16\"]]: Missing synchronization.",
```

Comment:
>[[\"SKIP_CHAR_BUFFER\"|\"relative://src/main/java/org/apache/commons/io/IOUtils.java:169:6517:16\"]] [](start = 68, length = 100)

looks like Semmle messages can encode links within the messages, an interesting concept we don't provide. The SARIF analog here would be to convert this location into a related location. Probably the right thing to do is to grab the section of the message that is effectively the text string for the link, here, "SKIP_CHAR_BUFFER" and to convert it into a standard message, stuffed into related location. This suggestion only makes sense if this Semmle convention is restricted to provide links to declared symbols (we should ask about whether they use this linking strategy for arbitrary/other purposes)

"message": "\"SKIP_CHAR_BUFFER\" is declared here.",